src/state/thunkMiddleware.ts

```diff
@@ -93,14 +96,15 @@ export const requestExtensionDataTypes = () => {
   };
 };
 
+// FETCH ALL PRIMITIVE AND COMPLEX TYPES AND DISPATCH TWO TREES OF IMPLEMENTED TREE TYPES
```

Comment:
YOU DONT HAVE TO SHOUT I'M RIGHT THERE I HEAR YOU FINE!!!

Reply:
_ᶦ'ᵐ ʳᵉᵃˡˡʸ ˢᵒʳʳʸ_